source/assets/stylesheets/all.scss

```diff
@@ -17,6 +17,22 @@
 @import "components/portfolio";
 @import "components/rubyshop";
 
+#events {
+  .timeline-image {
```

Comment:
It doesn't _need_ to. But I style like I code. 😜 

More seriously, I usually keep it like that, because it makes things easier when I'm constantly adding and removing rules. And it's compiled to the compact form anyway.